providers/smartlead/objectNames.go

```diff
@@ -0,0 +1,19 @@
+package smartlead
+
+import "github.com/amp-labs/connectors/common/handy"
+
+const (
+	objectNameCampaign     = "campaigns"
+	objectNameEmailAccount = "email-accounts"
+	objectNameClient       = "client"
+)
+
+var supportedObjectsByWrite = handy.NewSet([]string{ //nolint:gochecknoglobals
```

Comment:
were these only requirements? not lead? I don't have knowledge on the requirements myself.

Reply:
I got confused with instantly because this is instantly PR, so I thought there should be `lead`.

Reply:
Smartlead and Instantly have related set of Objects but with some differences.